realgud/common/cmds.el

```diff
@@ -291,7 +291,14 @@ EVENT should be a mouse click on the left fringe or margin."
 (defun realgud:cmd-eval-at-point()
   "Eval symbol under point."
   (interactive)
-  (realgud:cmd-run-command (thing-at-point 'symbol) "eval"))
+
+  (beginning-of-thing 'symbol)
```

Comment:
I needed to add `(require 'thingatpt)` to get this to work.

Reply:
huh, i guess all these functions have been required by other packages i use so i don't notice that i need to require them.

Reply:
Yeah, these things happen. Another reason why he tests are great - they work in an minimal environment with nothing extra loaded. 

Let's just add the require and then I think we can merge this. The other stuff I guess will have to wait. I am not sure how to update this pull request short of creating another branch. Or I can merge and then add, but it would be cleaner if you did it.

Reply:
i pushed the change

Reply:
I don't understand. What this change address? You still need a `(require 'thingatpt)` right?: 

```
 $ emacs --batch -q --eval '(beginning-of-thing)' 
...
 Symbol’s function definition is void: beginning-of-thing
```